tools/mmp/driver.cs

```diff
@@ -358,6 +367,7 @@ static void Main2 (string [] args)
 				{ "xamarin-framework-directory=", "The framework directory", v => { xm_framework_dir = v; }, true },
 				{ "xamarin-full-framework", "Used with --target-framework=4.5 to select XM 4.5 Target Framework", v => { IsUnifiedFullXamMacFramework = true; } },
 				{ "xamarin-system-framework", "Used with --target-framework=4.5 to select XM 4.5 Target Framework", v => { IsUnifiedFullSystemFramework = true; } },
+				{ "disable-partial-static-registrar", "Disable use of partial static registrar.", v => disable_partial_static_registrar = true },
```

Comment:
We could @rolfbjarne but my concern is that we currently mutate that value in:

```
static void Pack (IList<string> unprocessed)
{
              ... SNIP ...
			if (registrar == RegistrarMode.Default)
				registrar = App.EnableDebug ? RegistrarMode.Dynamic : RegistrarMode.Static;
			if (is_extension)
				registrar = RegistrarMode.Static;
```

And I didn't want to either stop doing that (and find every place that depended on it for now).